cmd/dp-observation-importer/main.go

```diff
@@ -81,7 +81,7 @@ func main() {
 
 	// objects to get dimension data - via the dataset API + cached locally in memory.
 	httpClient := http.Client{Timeout: time.Second * 15}
-	dimensionStore := dimension.NewStore(config.DatasetAPIURL, config.DatasetAPIAuthToken, &httpClient)
+	dimensionStore := dimension.NewStore(config.ServiceAuthToken, config.DatasetAPIURL, config.DatasetAPIAuthToken, &httpClient)
```

Comment:
Recommend changing `config` variable to be `cfg` to avoid conflicts with the package name